libraries/helpers.rb

```diff
@@ -93,6 +93,15 @@ def user_has_password?(new_resource)
       res.stdout =~ /1 row/ ? true : false
     end
 
+    def attribute_is_set?(user, attr, value)
+      sql = %(SELECT rolconfig FROM pg_roles WHERE rolname='#{user}';)
+      cmd = psql_command_string(new_resource, sql)
+
+      res = execute_sql(new_resource, cmd)
+      puts "AAAAAAAAA #{res.stdout}"
```

Comment:
Is this still needed? Seems like debug testing.

Reply:
lol, oops, i definately dont need that